node/pkg/vaa/structs.go

```diff
@@ -91,93 +135,35 @@ func (a SignatureData) String() string {
 }
 
 func (c ChainID) String() string {
-	switch c {
-	case ChainIDUnset:
-		return "unset"
-	case ChainIDSolana:
-		return "solana"
-	case ChainIDEthereum:
-		return "ethereum"
-	case ChainIDTerra:
-		return "terra"
-	case ChainIDBSC:
-		return "bsc"
-	case ChainIDPolygon:
-		return "polygon"
-	case ChainIDAvalanche:
-		return "avalanche"
-	case ChainIDOasis:
-		return "oasis"
-	case ChainIDAurora:
-		return "aurora"
-	case ChainIDFantom:
-		return "fantom"
-	case ChainIDAlgorand:
-		return "algorand"
-	case ChainIDEthereumRopsten:
-		return "ethereum-ropsten"
-	case ChainIDKarura:
-		return "karura"
-	case ChainIDAcala:
-		return "acala"
-	case ChainIDKlaytn:
-		return "klaytn"
-	case ChainIDCelo:
-		return "celo"
-	case ChainIDMoonbeam:
-		return "moonbeam"
-	case ChainIDNeon:
-		return "neon"
-	case ChainIDTerra2:
-		return "terra2"
-	default:
+	chainString := chainIntStringMap[int(c)]
+
+	if chainString == "" {
```

Comment:
Is it better to do "chainString, exists := chainIntStringMap[int(c)]" an check exists, rather than checking for the returned value being empty?

Also, you don't need the else below.

Reply:
Yes, your approach is the most idiomatic go.